System/Directory/OsPath.hs

```diff
@@ -52,6 +52,7 @@ module System.Directory.OsPath
     , copyFile
     , copyFileWithMetadata
     , getFileSize
+    , replaceFile
```

Comment:
I think the convention is to provide the same set of functions both from `System.Directory` and from `System.Directory.OsPath`.

Reply:
Done